packages/openapi-v2/src/controller-spec.ts

```diff
@@ -167,6 +174,25 @@ function resolveControllerSpec(constructor: Function): ControllerSpec {
 
     debug(`  adding ${endpointName}`, operationSpec);
     spec.paths[path][verb] = operationSpec;
+
+    debug(`  inferring schema definition for method %s`, op);
+    const paramTypes = MetadataInspector.getDesignTypeForMethod(
+      constructor.prototype,
+      op,
+    ).parameterTypes;
+
+    for (const p of paramTypes) {
+      if (
+        !includes([String, Number, Boolean, Array, Object], p) &&
```

Comment:
Please extract the multi-line condition into a named variable, see http://loopback.io/doc/en/contrib/style-guide.html#indentation-of-multi-line-expressions-in-if